migrator/postgreshelper/utils_test.go

```diff
@@ -107,3 +108,33 @@ func (s *PostgresRestoreSuite) TestUtilities() {
 	// Make sure connection to active database was terminated
 	s.NotNil(activePool.Ping(s.ctx))
 }
+
+func (s *PostgresRestoreSuite) TestWrapRollbackErrorPath() {
```

Comment:
Thinking of this a bit more, we can test both paths in one function so it can be renamed.

```suggestion
func (s *PostgresRestoreSuite) TestWrapRollback() {
```